genePred/genePred_test.go

```diff
@@ -27,8 +27,9 @@ var ReadTests = []struct {
 func TestRead(t *testing.T) {
 	for _, test := range ReadTests {
 		actual := Read(test.name)
-		if !AllAreEqual(test.data, actual) {
-			t.Errorf("The %s file was not read correctly.", test.name)
+		equal, fields := AllAreEqual(test.data, actual)
```

Comment:
Do we have a check where we compare two lists of GenePreds where one of them are not equal and "not equal" is the correct answer?